modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -166,41 +134,46 @@ code) and the Market app is enabled, but there is no available internet
 connection, then these apps will need to be manually updated once the
 upgrade is finished.
 
+[[copy-old-apps]]
+== Copy Old Apps
+If you are using third party or enterprise applications, look in your new `/var/www/owncloud/apps/` directory to see if they are there.
+If not, copy them from your old `apps/` directory to your new one, and make sure that the directory permissions are the same as for the other ones.
```

Comment:
please just continue writing, it is not necessary to do a linebreak like that one. You can do a lb at about 80 chars just for better readbility in your editor, but this here does not look good.
`permissions are the same as for the other ones` --> `permissions match the existing ones`